### DISCUSSION TITLE
GitHub Teacher Pack

### DISCUSSION THREAD

**Discussion by aqillakhamis** [community/community#120523] [GitHub Education]
### Hi everyone,

Hi everyone, I am unable to get teacher benefit even my academic profile has been verified. Can someone help me to figure out why this happen even all the steps required I have completed

**Comment by samus-aran** [comment]
Hi @aqillakhamis  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by aqillakhamis** [reply]
![image](https://github.com/community/community/assets/28037151/b8739dd5-361c-46ee-9e1d-b248e7227693)

my application not got rejected, but when I tried to select my school for teacher pack benefit. I am unable to click the continue button. All the requirements needed to apply for this teacher benefit I have fulfilled. I am glad if you can help me on this matter

Thank you! 😃
